math/kissfft/CMakeLists.txt

```diff
@@ -0,0 +1,74 @@
+# ##############################################################################
+# apps/math/kissfft/CMakeLists.txt
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more contributor
+# license agreements.  See the NOTICE file distributed with this work for
+# additional information regarding copyright ownership.  The ASF licenses this
+# file to you under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License.  You may obtain a copy of
+# the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+# License for the specific language governing permissions and limitations under
+# the License.
+#
+# ##############################################################################
+
+if(CONFIG_MATH_KISSFFT)
+
+  # ############################################################################
+  # Config and Fetch kissfft lib
+  # ############################################################################
+
+  set(KISSFFT_DIR ${CMAKE_CURRENT_LIST_DIR}/kissfft)
+
+  if(NOT EXISTS ${KISSFFT_DIR})
+    set(KISSFFT_URL
+        https://github.com/mborgerding/kissfft/archive/refs/tags/v130.zip)
```

Comment:
Should version be exposed as configuration?

Reply:
ditto